velox/exec/tests/utils/PlanBuilder.cpp

```diff
@@ -293,6 +293,42 @@ PlanBuilder& PlanBuilder::filter(const std::string& filter) {
   return *this;
 }
 
+PlanBuilder& PlanBuilder::tableWrite(const std::string& outputDirectoryPath) {
+  auto rowType = planNode_->outputType();
+
+  std::vector<std::shared_ptr<const connector::hive::HiveColumnHandle>>
+      columnHandles;
+  for (auto i = 0; i < rowType->size(); ++i) {
+    columnHandles.push_back(std::make_shared<connector::hive::HiveColumnHandle>(
+        rowType->nameOf(i),
+        connector::hive::HiveColumnHandle::ColumnType::kRegular,
+        rowType->childAt(i),
+        rowType->childAt(i)));
+  }
+
+  auto locationHandle = std::make_shared<connector::hive::LocationHandle>(
+      outputDirectoryPath,
+      outputDirectoryPath,
+      connector::hive::LocationHandle::TableType::kNew);
+  auto hiveHandle = std::make_shared<connector::hive::HiveInsertTableHandle>(
+      columnHandles,
+      locationHandle,
+      dwio::common::FileFormat::DWRF,
```

Comment:
Can we make this an argument defaulted to `dwio::common::FileFormat::DWRF`?
Will be easier to test the Parquet format.

Reply:
@majetideepak Are these existing tests that would benefit from this change? If not, maybe do this when we have a need?

Reply:
https://github.com/facebookincubator/velox/blob/main/velox/connectors/hive/storage_adapters/s3fs/tests/S3InsertTest.cpp#L75 that can benefit from this change.
But I can take this up in a separate PR.

Reply:
Got it. I'll leave it you for a follow-up then. Thanks.